doc/performance-testing/cloud-run/0001_stress-test-december-2021.md

```diff
@@ -0,0 +1,86 @@
+# Stress Test December 2021
+This is an initial stress test to asses the performance of runner after the switch from kubernetes to cloud run. Below are the initial settings we used:
+
+
+## Runner settings
+
+| Setting | Value |
+| --- | ---| 
+| Concurrency | 250 |
+| Max instances   | 1 |
+| Min instances | 1 |
+
+
+## Benchmark settings
+
+| Setting | Value |
+| --- | ---| 
+| Load injector instances | 1 |
+| Clients hatch rate   | 50 |
+| Wait time minimum | 1 |
+| Wait time maximum | 2 |
+| Requests JSON | test_benchmark_business.json |
+| Runtime | 10m |
+
+## Results
+
+| Clients per instance | Requests per second | 99th percentile Max CPU Usage (%) | 99th percentile response time (ms) | Error rate (%) | Output |
+| --- | --- | --- | --- | --- | --- |
+| 50  | 59.63 | 32.0  | 140 | 0.000 | [output](https://console.cloud.google.com/storage/browser/eq-stress-injector-07122021-outputs/stress-test/2021-12-08T09:23:30) |
```

Comment:
Do we usually link to GCP or keep it on Github? I forget

Reply:
We copied the format from the Kubernetes stress tests which link to GCP 👍

Reply:
Ok, I couldn't remember